src/web/app/pages-student/student-home-page/student-home-page.component.spec.ts

```diff
@@ -36,4 +36,60 @@ describe('StudentHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with null fields', () => {
```

Comment:
`null` -> `default`. The fields all have default values, as you can see in `student-home-page.component.ts`.